### PR TITLE
docs(storybook): add version button in storybook toolbar

### PR DESCRIPTION
**Pull request by ozkersemih**
In this pr, I want to add custom toolbar button in storybook to redirect beta/stable version
Screenshot 2024-03-04 at 17 21 34

### CODE REVIEW

**Review by Enes5519 — APPROVED** [review]

File: .storybook/preview.ts [hunk 1]
```diff
@@ -29,3 +29,31 @@ export const parameters = {
...
+
+export const globalTypes = {
+  version: {
+    name: 'version',
+    description: 'Select version Stable/Beta',
+    toolbar: {
+      icon: 'branch',
+      items: [
+        { value: 'stable', icon: 'branch', title: 'Stable Version' },
+        { value: 'beta', icon: 'branch', title: 'Beta(next) Version' },
```
Comment (erbilnas) [inline .storybook/preview.ts:41]: We don't need to include 'Next' in parentheses; 'Beta Version' is sufficient.

**Review by erbilnas — CHANGES_REQUESTED** [review]

**Comment by ozkersemih** [comment]
> I don't think it is visible enough within the UI. Is that possible to place a select component or similar under the search input here?
> 
> image

I totally aggre with you but i didn't find anything about to add selection for version switching in storybook.

File: .storybook/preview.ts (hunk 1 above)
Comment (ozkersemih) [reply .storybook/preview.ts]: ok, i changed

**Review by leventozen — APPROVED** [review]
LGTM!

**Review by AykutSarac — APPROVED** [review]

**Review by erbilnas — APPROVED** [review]

**Comment by buseselvi** [comment]
Great job! Let's add a button like the one below on the welcome pages, too. 

image

**Review by AykutSarac — CHANGES_REQUESTED** [review]
@erbilnas, in reference to your recent commit, Baklava CDN is integrated into the storybook. However, since the README.md is also displayed on the GitHub repository, we won't be able to access Baklava components from there.

**Comment by ozkersemih** [comment]
@AykutSarac @erbilnas I have another idea
```
// welcome.stories.mdx file
<Markdown>{ReadMe.slice(0,ReadMe.search("## How to use"))}</Markdown>
<bl-button>...</bl-button>
<Markdown>{ReadMe.slice(ReadMe.search("## How to use"))}</Markdown>
```
We can split readme in welcome mdx file like this. With this way, also we wont crash original readme file. What do u think?

**Comment by erbilnas** [comment]
> @erbilnas, in reference to your recent commit, Baklava CDN is integrated into the storybook. However, since the README.md is also displayed on the GitHub repository, we won't be able to access Baklava components from there.

You're correct, let me amend it now.

**Comment by ozkersemih** [comment]
Last touch will be like below, I think it is ok.
```
<Meta title="Documentation/Welcome" />

<Markdown>{ReadMe.slice(0,ReadMe.search("Baklava is a design system"))}</Markdown>
<div align={"center"}>
  {window.parent.location.hostname.includes('next') && <bl-button variant="secondary" size="large" onclick="window.open('https://baklava.design/', '_blank')">Stable Version</bl-button>}
  {!window.parent.location.hostname.includes('next') && <bl-button variant="secondary" size="large" onclick="window.open('https://next.baklava.design/', '_blank')">Beta Version</bl-button>}
</div>
<Markdown>{ReadMe.slice(ReadMe.search("Baklava is a design system"))}</Markdown>
```
Screenshot 2024-03-18 at 16 18 14

**Review by AykutSarac — APPROVED** [review]

**Pull request merged**